src/metrics/rules/active_ruleset.go

```
@@ -296,12 +318,12 @@ func (as *activeRuleSet) rollupResultsFor(id []byte, timeNanos int64, matchOpts
 			tags = append(tags, snapshot.tags)
 		}
 	}
-	// NB: could log the matching error here if needed.
-	res, _ := as.toRollupResults(id, cutoverNanos, rollupTargets, keepOriginal, tags, matchOpts)
```

We were swallowing this error before which seems like a bad idea?